apps/web/src/design-system/table/Table.tsx

```diff
@@ -123,13 +125,15 @@ export function Table({
 
             return (
               <tr
+                key={row.id}
```

Comment:
same about the row, keys are auto-generated

Reply:
Fixed